### PR TITLE
Refactored duplicate diff methods in color.rb and html.rb

### PR DESCRIPTION
**Pull request by doolin**
Noticed some duplicated code so refactored it. Passes all specs in spec/shell. Naming could be better, willing to rework that.

### CODE REVIEW

File: lib/thor/shell/color.rb [hunk 1]
```diff
@@ -102,45 +106,18 @@ def can_display_colors?
...
-            actual  = File.binread(destination).to_s.split("\n")
-            content = content.to_s.split("\n")
-
-            Diff::LCS.sdiff(actual, content).each do |diff|
-              output_diff_line(diff)
-            end
-          else
-            super
-          end
+          show_diff_common(destination, content)
```
Comment (rafaelfranca) [inline lib/thor/shell/color.rb]: :+1:

**Comment by doolin** [comment]
I'll take a look at this in a day or two, get it rebased, etc. Thanks for taking a look!

**Comment by rafaelfranca** [comment]
Merged in f3d5d19. Thanks

**Comment by doolin** [comment]
Excellent, thank you.

**Pull request closed without merging**